StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1000KeywordsMustBeSpacedCorrectly.cs

```diff
@@ -86,25 +98,28 @@ private static void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
                 case SyntaxKind.LetKeyword:
                 case SyntaxKind.LockKeyword:
                 case SyntaxKind.OrderByKeyword:
+                case SyntaxKind.OutKeyword:
+                case SyntaxKind.RefKeyword:
                 case SyntaxKind.SelectKeyword:
                 case SyntaxKind.StackAllocKeyword:
                 case SyntaxKind.SwitchKeyword:
                 case SyntaxKind.UsingKeyword:
+                case SyntaxKind.TypeVarKeyword:
```

Comment:
❗️ this is the `typevar` keyword used as an attribute location, not the `var` keyword I intended.